requirements/requirements.in

```diff
@@ -1,28 +1,29 @@
 apache-libcloud==2.0.0
 appdirs==1.4.2
-asgi-amqp==0.4.1
-asgiref==1.0.1
+asgi-rabbitmq==0.5.4
+asgiref==1.1.2
 azure==2.0.0rc6
 backports.ssl-match-hostname==3.5.0.1
 boto==2.46.1
 boto3==1.4.4
-channels==0.17.3
-celery==3.1.17
-daphne>=0.15.0,<1.0.0
-Django==1.8.16
+channels==1.1.8
+celery==4.1
+daphne==1.3.0
+Django==1.11.7
```

Comment:
Should we continue to pin exact versions in `requirements.in`, since that's really what `requirements.txt` does? Specifying `Django<2.0` would automatically pull in bug fix releases via `pip-compile` without having to check each of these dependencies manually.

Reply:
👍